app/react/Metadata/components/MetadataFormFields.js

```diff
@@ -120,6 +148,7 @@ export class MetadataFormFields extends Component {
             options={thesauri}
             totalPossibleOptions={totalPossibleOptions}
             prefix={_model}
+            onChange={this.relationshipChange.bind(this, property)}
```

Comment:
Please check this eslint warning. The recommended approach is to do the binding in the class constructor.

Reply:
How do you suggest doing the binding in the class constructor but also bind different properties?